src/components/ExpandedNode/__tests__/ExpandedNode.test.tsx

```diff
@@ -89,12 +89,12 @@ const testMedicationRequestState: GuidanceState = {
 };
 
 describe('<ExpandedNode />', () => {
-  it('renders a ExpandedNode for action state', () => {
+  it('renders a ExpandedNode for action node', () => {
     const { getByText, queryByRole, queryByText } = render(
       <ExpandedNode
-        pathwayState={testActionState}
+        actionNode={testActionState}
```

Comment:
minor, but I imagine this was meant to be renamed as well.

```suggestion
        actionNode={testActionNode}
```